packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -547,8 +547,8 @@ function computeShadowMode(
     return shadowMode;
 }
 
-function assertIsVM(obj: any): asserts obj is VM {
-    if (isNull(obj) || !isObject(obj) || !('renderRoot' in obj)) {
+function assertIsVM(obj: unknown): asserts obj is VM {
```

Comment:
`assertIsVM` is only used in non-prod mode, so this is low-risk FWIW.

Reply:
Changing a parameter from `any` to `unknown` doesn't impact the external usage. Both are function as catch-all types.